calcom/cal.com
# Fixed text beyond X icon issue

## What does this PR do?

Fixes #8976 

 Loom Video: https://www.loom.com/share/0245445580704f2798834808d005b9d8

**Environment**: Production

## Type of change

Bug fix (non-breaking change which fixes an issue)
Fixed text to not go behind the cross icon if the cross icon is visible which happens only when options are > 2.

## How should this be tested?

Make sure you can type text upto the whole input field and it doesn't go behind the cross icon
